.travis.yml

```diff
@@ -5,10 +5,20 @@ sudo: required
 services:
     - docker
 
+cache:
+    directories:
+        - "${HOME}/google-cloud-sdk/"
+
 before_install:
+    # Workaround to make gsutil work (see travis-ci#7940).
+    - sudo rm -f /etc/boto.cfg
```

Comment:
"export BOTO_CONFIG=/dev/null" is another workaround also referenced in travis-ci#7940

Reply:
I've verified the current fix, and successfully use gsutil to upload log. Let's use the verified one now. :)

Reply:
Added TODO to remove it.